specification/sql/resource-manager/Microsoft.Sql/2017-03-01-preview/managedDatabases.json

```diff
@@ -1,556 +1,726 @@
+            "properties": {
+                "collation": {
+                    "description": "Collation of the managed database.",
+                    "type": "string",
+                    "x-ms-mutability": [
+                        "read",
+                        "create"
+                    ]
+                },
+                "status": {
+                    "description": "Status for the database.",
+                    "enum": [
+                        "Online",
+                        "Offline",
+                        "Shutdown",
+                        "Creating",
+                        "Inaccessible"
+                    ],
+                    "type": "string",
+                    "readOnly": true,
+                    "x-ms-enum": {
+                        "name": "ManagedDatabaseStatus",
+                        "modelAsString": true
+                    }
+                },
+                "creationDate": {
+                    "format": "date-time",
+                    "description": "Creation date of the database.",
+                    "type": "string",
+                    "readOnly": true
+                },
+                "defaultSecondaryLocation": {
+                    "description": "Geo paired region.",
+                    "type": "string",
+                    "readOnly": true
+                },
+                "catalogCollation": {
+                    "description": "Collation of the metadata catalog.",
+                    "enum": [
+                        "DATABASE_DEFAULT",
+                        "SQL_Latin1_General_CP1_CI_AS"
+                    ],
+                    "type": "string",
+                    "x-ms-enum": {
+                        "name": "CatalogCollationType",
+                        "modelAsString": true
+                    },
+                    "x-ms-mutability": [
+                        "read",
+                        "create"
+                    ]
+                },
+                "createMode": {
+                    "description": "Managed database create mode. PointInTimeRestore: Create a database by restoring a point in time backup of an existing database. SourceDatabaseName, SourceManagedInstanceName and PointInTime must be specified. RestoreExternalBackup: Create a database by restoring from external backup files. Collation, StorageContainerUri and StorageContainerSasToken must be specified.",
+                    "enum": [
+                        "Default",
+                        "RestoreExternalBackup",
+                        "PointInTimeRestore"
+                    ],
+                    "type": "string",
+                    "x-ms-enum": {
+                        "name": "ManagedDatabaseCreateMode",
+                        "modelAsString": true
+                    },
+                    "x-ms-mutability": [
+                        "create"
+                    ]
+                },
+                "lastBackupName": {
```

Comment:
Move to correct object

Reply:
This is not supposed to be inside any of the create requests, this is used for complete restore request.